app/src/main/java/org/gdg/frisbee/android/activity/PulseActivity.java

```diff
@@ -65,8 +72,6 @@ public void onCreate(Bundle savedInstanceState) {
         Timber.i("onCreate");
         setContentView(R.layout.activity_pulse);
 
-        getSupportActionBar().setLogo(R.drawable.ic_logo_pulse);

```

Comment:
What should we do with these logos, I remember spesifically this logo was good. :)